packages/go/ein/azure.go

```diff
@@ -41,6 +42,8 @@ const (
 	KeyVaultPermissionGet string = "Get"
 )
 
+var resourceGroupLevel, _ = regexp.Compile("^[\\w\\d\\-\\/]*/resourceGroups/[0-9a-zA-Z]+$")
```

Comment:
One last nit, apologies. Please rewrite `regexp.Compile` to `regexp.MustCompile` as the latter will force a panic if the regex is invalid.

Reply:
Makes sense :)